rc/filetype/markdown.kak

```diff
@@ -45,12 +45,12 @@ evaluate-commands %sh{
     awk c cabal clojure coffee cpp crystal css cucumber d diff dockerfile elixir erlang fish
     gas go haml haskell html ini java javascript json julia kak kickstart
     latex lisp lua makefile markdown moon objc ocaml perl pug python ragel
-    ruby rust sass scala scss sh swift toml tupfile typescript yaml sql
+    ruby rust sass scala scss sh swift toml ts tupfile typescript yaml sql
   "
   for lang in ${languages}; do
     printf 'add-highlighter shared/markdown/%s region -match-capture ^(\h*)```\h*(%s\\b|\\{[.=]?%s\\})   ^(\h*)``` regions\n' "${lang}" "${lang}" "${lang}"
     printf 'add-highlighter shared/markdown/%s/ default-region fill meta\n' "${lang}"
-    [ "${lang}" = kak ] && ref=kakrc || ref="${lang}"
+    [ "${lang}" = kak ] && ref=kakrc || [ "${lang}" = ts ] && ref=typescript || ref="${lang}"
```

Comment:
Unfortunately this will cause unexpected results if `$lang = kak` (it will set ref=typescript). Consider

	$ true && echo kakrc || false && echo typescript || echo fallthrough
	kakrc
	typescript

Looks like the jobs separated by && and || are evaluated one-by-one, left-to-right.
How about using a `case` statement instead?

Reply:
Added a case statement -- Done!